File: examples/example-1/export-single-chart.js
// Import FusionExport SDK client for Node.js
const {
    ExportManager,
    ExportConfig
} = require('fusionexport-node-client');

// Instantiate ExportConfig and ExportManager
const exportConfig = new ExportConfig();
const exportManager = new ExportManager();
let template;

// Create a simple HTML to export
template = '<h1>Hello world</h2>';

// Set all the configurations
exportConfig.set('template', template);
exportConfig.set('type', 'pdf');
exportConfig.set('templateFormat', 'a4');
exportConfig.set('asyncCapture', true);

// Let's export!
exportManager.export(exportConfig, outputDir = '.', unzip = true).then((exportedFiles) => {
    exportedFiles.forEach(file => console.log(file));
}).catch((err) => {
    console.log(err);
});

File: python/examples/example-1/export_single_chart.py
# Import FusionExport SDK client for Python
from fusionexport import ExportManager, ExportConfig
import os

# Instantiate ExportConfig and ExportManager
export_config = ExportConfig()
export_manager = ExportManager()

# Fetch the curr directory of the template
curr_dir = os.path.dirname(os.path.abspath(__file__))
template_path = os.path.join(curr_dir, 'template.html')

# Set all the configurations
export_config['templateFilePath'] = template_path
export_config['type'] = 'pdf'
export_config['templateFormat'] = 'A4'
export_config['asyncCapture'] = True

# Export your first PDF
export_manager.export(export_config, unzip = True)
